python/ray/dataframe/dataframe.py

```diff
@@ -977,9 +978,34 @@ def assign(self, **kwargs):
             "github.com/ray-project/ray.")
 
     def astype(self, dtype, copy=True, errors='raise', **kwargs):
-        raise NotImplementedError(
-            "To contribute to Pandas on Ray, please visit "
-            "github.com/ray-project/ray.")
+        if errors == 'raise':
```

Comment:
Should not ignore errors when `errors == 'raise'`

Reply:
@kunalgosar What do you mean by your comment?

Reply:
On `errors == 'raise'`, these exceptions should be raised. On `errors == 'ignore'`, they should be suppressed and `self` should be returned as per the pandas documentation.